client/styles/components/_nav.scss

```diff
@@ -76,7 +76,7 @@
 		display: flex;
 		position: absolute;
 		flex-direction: column;
-  	top: 4px;
+  	top: 40px;
```

Comment:
Indent it properly

Reply:
Not Sure , its showing perfectly in my code editor , maybe github diff loader is showing it this way?

Reply:
> Indent it properly
Not Sure , its showing perfectly in my code editor , maybe github diff loader is showing it this way?

Reply:
also this should be a rem measurement, not a pixel measurement.

Reply:
This was already a pixel measurement , should I change it?

Reply:
yes!